test/enhance/test_adjust.py

```diff
@@ -637,6 +637,71 @@ def test_gradcheck_accumulative(self, device, dtype):
                          raise_exception=True)
 
 
+class TestAdjustSigmoid:
+    @pytest.mark.parametrize("shape", [(3, 4, 4), (2, 3, 4, 4)])
+    def test_shape_sigmoid(self, shape, device):
+        inputs = torch.ones(*shape, device=device)
+        f = kornia.enhance.adjust_sigmoid
+        assert f(inputs).shape == torch.Size(shape)
+
+    def test_sigmoid(self, device, dtype):
+        data = torch.tensor(
+            [
+                [[[1.0, 1.0], [1.0, 1.0]], [[0.5, 0.5], [0.5, 0.5]], [[0.25, 0.25], [0.25, 0.25]]],
+                [[[0.0, 0.0], [0.0, 0.0]], [[0.3, 0.3], [0.3, 0.3]], [[0.6, 0.6], [0.6, 0.6]]],
+            ],
+            device=device,
+            dtype=dtype,
+        )  # 2x3x2x2
+
+        expected = torch.tensor(
+            [
+                [[[0.9933, 0.9933], [0.9933, 0.9933]], [[0.5, 0.5], [0.5, 0.5]],
+                 [[0.07585, 0.07585], [0.07585, 0.07585]]],
+                [[[0.00669, 0.00669], [0.00669, 0.00669]], [[0.1192, 0.1192], [0.1192, 0.1192]],
+                 [[0.7310, 0.7310], [0.7310, 0.7310]]],
+            ],
+            device=device,
+            dtype=dtype,
+        )  # 2x3x2x2
+
+        f = kornia.enhance.AdjustSigmoid()
+        assert_close(f(data), expected)
```

Comment:
can you add gradcheck, and/or onxx-jit tests ?

Reply:
Added, can you check? I suppose that all such tests should be marked with specific marks.

Reply:
marks are not really required